net/respondd-module-wifisettings/README.md

```diff
@@ -0,0 +1,13 @@
+This module adds a respondd wifisettings usage nodeinfo provider.
+The format is the following:
+
+```json
+{
+  "nodeinfo": {
+    "channel24": 11,
+    "txpower24": 22,
+    "channel5": 44,
+    "txpower5": 18,
```

Comment:
How will multiple radios in the same band be handled.

Reply:
I think having an array of devices would make sense, each with attributes band, channel, and txpower. htmode might be another interesting piece of information.